spec/lib/availability_response_spec.rb

```diff
@@ -19,60 +19,68 @@
       expect(availability.keys).to eql %w{991023558879703811 991036880906903811}
     end
 
-    it "has the exepected value" do
-    expected =
-      { "991023558879703811" =>
-       { holdings:
-        [{
-        "holding_id" => "22277782420003811",
-        "institution" => "01TULI_INST",
-        "library_code" => "MAIN",
-        "location" => "Stacks",
-        "call_number" => "NX511.N4 N513 2006",
-        "availability" => "available",
-        "total_items" => "2",
-        "non_available_items" => "0",
-        "location_code" => "stacks",
-        "call_number_type" => "0",
-        "priority" => "1",
-        "library" => "Paley Library",
-        "inventory_type" => "physical" },
-         {
-        "holding_id" => "22277782440003811",
-        "institution" => "01TULI_INST",
-        "library_code" => "AMBLER",
-        "location" => "Stacks",
-        "call_number" => "NX511.N4 N513 2006",
-        "availability" => "available",
-        "total_items" => "1",
-        "non_available_items" => "0",
-        "location_code" => "stacks",
-        "call_number_type" => "0",
-        "priority" => "2",
-        "library" => "Ambler Campus Library",
-        "inventory_type" => "physical" }] },
-     "991036880906903811" =>
-      { holdings:
-        [{
-        "holding_id" => "22413618700003811",
-        "institution" => "01TULI_INST",
-        "library_code" => "MAIN",
-        "location" => "Stacks",
-        "call_number" => "PN4775 .F45 2019",
-        "availability" => "available",
-        "total_items" => "1",
-        "non_available_items" => "0",
-        "location_code" => "stacks",
-        "call_number_type" => "0",
-        "priority" => "1",
-        "library" => "Paley Library",
-        "inventory_type" => "physical",
-        "holding_info" => "holding-1 holding-2 holding-3" }]
+    it "skips the empty holding info (code=t) subfield" do
+      expect(availability["991023558879703811"][:holdings][0]["holding_info"]).to be_nil
```

Comment:
@dbwwei The format of the fixture for `991023558879703811` doesn't match what you are testing for. It doesn't have a subfield with code=t. I like that we have this test as a base case for records that don't have holding info included.

There should be an additional fixture that includes a subfield code=t but has no content inside it.

Other scenarios could be a record with a single subfield code=t with content in it and a record with multiple subfields code=t where one of them is missing content.

It should be possible to add more fixture records in the [multiple_bibs.json](https://github.com/tulibraries/alma_rb/blob/b291f2489ec674dbde8475a0da4c45a3896072c0/spec/fixtures/multiple_bibs.json) to capture these additional scenarios. Make sure they have distinct IDs. Also since MARCXML string is hard to read in the "anies" field, could you try formatting the string to span across multiple lines in the file (does it work if you add `\n` at the ends of lines?) and only including the essential bits of the record (removing all of the datafields except the 001 and the AVA tag) so it's easier to tell what is actually in the fixture data?